File: Data Structures & Algorithms/rotate-array/submission-5.py
class Solution:
    def rotate(self, nums: List[int], k: int) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        nums.reverse()

        while k:
            num = nums.pop(0)
            nums.append(num)
            k -= 1
        
        nums.reverse()

File: Data Structures & Algorithms/remove-element/submission-5.cs
public class Solution {
    public int RemoveElement(int[] nums, int val) {
        int k = 0;
        int length = nums.Length;
        for (int i = 0; i < length; i++){
            if (nums[i] != val){
                if (i != k){
                    nums[k] = nums[i];
                }
                k++;
            }
        }
        return k;
    }
}